controllers/imagecollector/imagecollector_controller.go

```diff
@@ -42,19 +42,24 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
+	batchv1 "k8s.io/api/batch/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var (
-	collectorImage = flag.String("collector-image", "ghcr.io/azure/collector:latest", "collector image")
-	log            = logf.Log.WithName("controller").WithValues("process", "imagecollector-controller")
-	repeatPeriod   = flag.Duration("repeat-period", time.Hour*24, "repeat period for collect/scan process")
+	scannerImage           = flag.String("scanner-image", "", "scanner image")
+	collectorImage         = flag.String("collector-image", "", "collector image")
+	log                    = logf.Log.WithName("controller").WithValues("process", "imagecollector-controller")
+	repeatPeriod           = flag.Duration("repeat-period", time.Hour*24, "repeat period for collect/scan process")
+	deleteScanFailedImages = flag.Bool("delete-scan-failed-images", true, "whether or not to delete images for which scanning has failed")
+	scanDisabled           = true
```

Comment:
I think it's best not to add more mutable global state if we don't need to. We already have `scannerImage` so we don't need `scanDisabled` -- there's the slight potential for them to get out of sync with each other.